hack/build-images.sh

```diff
@@ -87,6 +90,27 @@ image openshift/hello-openshift              examples/hello-openshift
 docker build --no-cache -t openshift/deployment-example:v1 examples/deployment
 docker build --no-cache -t openshift/deployment-example:v2 -f examples/deployment/Dockerfile.v2 examples/deployment
 
+# images for integrated services (e.g. logging and metrics)
+clonedir="$(mktemp -d 2>/dev/null || mktemp -d -t clonedir.XXXXXX)"
+pushd $clonedir
+# logging
+git clone --depth 1 --branch ${OS_LOGGING_BRANCH} https://github.com/openshift/origin-aggregated-logging.git 
```

Comment:
@mwringe This change is to presumable just allow building and tagging the logging and metrics images in lock step with origin.  We should add the e2e test some place and I don't believe we have a card for it yet.  Presumably by the time we get to this point the images would have passed our tests.  Alternatively the buildj-images.sh output can be run without tagging and we could run e2e for testing.

@liggitt @smarterclayton These images are different in the build resources do not reside within the same repo like the other images.  Unless we are going to make some assumptions or possibly provide an ENV that points to the source location, I don't see any alternative to pulling down the source.  What changes would you like to see?